src/ui/layout/mct-tree.vue

```diff
@@ -14,61 +18,84 @@
     </div>
 
     <!-- search loading -->
-    <li
+    <div
         v-if="searchLoading && activeSearch"
         class="c-tree__item c-tree-and-search__loading loading"
     >
         <span class="c-tree__item__label">Searching...</span>
-    </li>
+    </div>
 
     <!-- no results -->
     <div
-        v-if="(searchValue && searchResultItems.length === 0 && !searchLoading)"
+        v-if="showNoSearchResults"
         class="c-tree-and-search__no-results"
     >
         No results found
     </div>
 
     <!-- main tree -->
-    <ul
+    <div
         ref="mainTree"
         class="c-tree-and-search__tree c-tree"
     >
         <!-- ancestors -->
-        <li v-if="!activeSearch">
+        <div v-if="!activeSearch">
+
+            <div
+                ref="dummyItem"
+                class="c-tree__item-h"
+                style="left: -1000px; position: absolute;"
```

Comment:
did you try using visibility: hidden ?

Reply:
I had not, but I just did and it will block the current first ancestor if I do that. I kept the styles as is but included visibility hidden as well as an added layer hiddenness (wow, that's a word). Another option would be to add position: relative to it and any other ancestors (or just the first one) and then use z-indexing to put it in the back.